src/ManagedCodecs/TargaCodec.cs

```diff
@@ -0,0 +1,131 @@
+﻿using System;
+using System.IO;
+using System.Runtime.CompilerServices;
+
+using SixLabors.ImageSharp;
+using SixLabors.ImageSharp.Advanced;
+using SixLabors.ImageSharp.PixelFormats;
+
+using PhotoSauce.MagicScaler;
+
+namespace PhotoSauce.ManagedCodecs
+{
+	/// <summary>Represents a TGA image file.</summary>
+	public sealed class TargaContainer : IImageContainer, IDisposable
+	{
+		private readonly Image decodedImage;
+
+		private TargaContainer(Image image) => decodedImage = image;
+
+		/// <inheritdoc />
+		public FileFormat ContainerFormat => FileFormat.Bmp;
+
+		/// <inheritdoc />
+		public int FrameCount => 1;
+
+		/// <inheritdoc />
+		public IImageFrame GetFrame(int index) => index == 0 ? new TargaFrame(decodedImage) : throw new IndexOutOfRangeException("Invalid frame index");
+
+		/// <summary>Loads a TGA image from the specified path.</summary>
+		/// <param name="imgPath">The path to the image file.</param>
+		/// <returns>A <see cref="TargaContainer"/> encapsulating the image.</returns>
+		public static TargaContainer Load(string imgPath)
+		{
+			int bpp = 0;
+			using (var fs = File.OpenRead(imgPath))
+			{
+				var info = Image.Identify(fs, out var fmt);
```

Comment:
It's a bit of a bummer that `Image.DetectFormat` and `Image.Load` have overloads that accept path but `Identify` only accepts a `Stream`.

Reply:
i agree, a overload for Identify with a path would be useful

Reply:
Issue opened: SixLabors/ImageSharp#1144

Reply:
Your wish is granted.
https://github.com/SixLabors/ImageSharp/pull/1146

Reply:
![zoltar](https://user-images.githubusercontent.com/7929102/76695696-ad9de380-663f-11ea-9d8b-896abd8b8bd1.gif)